plugins/wacom-i2c/wacom-i2c.quirk

```diff
@@ -0,0 +1,28 @@
+# AES bootloader mode
+[DeviceInstanceId=HIDRAW\VEN_056A&DEV_0094]
+Plugin = wacom_i2c
+Guid = WacomAES
+Flags = is-bootloader
+
+# EMR bootloader mode
+[DeviceInstanceId=HIDRAW\VEN_056A&DEV_012B]
+Plugin = wacom_i2c
+Guid = WacomEMR
+Flags = is-bootloader
+
+# Dell XPS-15
+[DeviceInstanceId=HIDRAW\VEN_056A&DEV_4875]
+Plugin = wacom_i2c
+Guid = WacomAES
```

Comment:
I suspect that is not going to be an appropriate GUID for Wacom - it's implying that the same payload works on "All" AES panels.  I don't know that's actually true.

Reply:
It's really only using the "fake" GUID to match up the `Flash` quirks. If there are AES panels with a different base-addr then we can certainly just put the quirks on the `DeviceInstanceId` section.

Reply:
I think a comment at the top of this file to explain to that effect would be useful for passerbys later.

Reply:
Done.